new-log-viewer/src/components/StatusBar/LogLevelSelect/LogLevelChip.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+    Chip,
+    Tooltip,
+} from "@mui/joy";
+import {DefaultColorPalette} from "@mui/joy/styles/types/colorSystem";
+
+import {LOG_LEVEL} from "../../../typings/logs";
+
+import "./LogLevelChip.css";
+
+
+/**
+ * Maps log levels to colors from JoyUI's default color palette.
+ */
+const LOG_LEVEL_COLOR_MAP: Record<LOG_LEVEL, DefaultColorPalette> = Object.freeze({
+    [LOG_LEVEL.UNKNOWN]: "neutral",
+    [LOG_LEVEL.TRACE]: "neutral",
+    [LOG_LEVEL.DEBUG]: "neutral",
```

Comment:
```suggestion
    [LOG_LEVEL.TRACE]: "success",
    [LOG_LEVEL.DEBUG]: "success",
```
or
<img width="306" alt="image" src="https://github.com/user-attachments/assets/6ed20fdc-8c3c-40c2-ba6e-7446833baba8">

Reply:
Sg